src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/Contracts/ExpandRequest.cs

```diff
@@ -57,6 +57,10 @@ public class ExpandParams
         /// Security token for AzureMFA authentication for refresing access token on connection.
         /// </summary>
         public SecurityToken? SecurityToken { get; set; }
+        ///<summary>
+        /// Filters to apply to the expand request
+        ///</summary>
+        public NodeFilter?[] Filters { get; set; }

```

Comment:
Shouldn't this be `NodeFilter[]?` This is currently saying "a required array of nullable NodeFilter items"